docs/esa/hubble.rst

```diff
@@ -89,8 +143,147 @@ It will also download a file storing all metadata for all observations
 associated with target name 'm31'. The result of the query will be stored in
 file 'm31_query.xml'.
 
+-----------------------------------------------------------------
+5. Querying observations by search criteria in the Hubble archive
+-----------------------------------------------------------------
+
+The query_by_criteria function uses a set of established parameters to create
+and execute an ADQL query, accessing the HST archive database usgin the Table
+Access Protocol (TAP). 
+
+PARAMETERS
+"""""""""""""""""
+- **calibration_level** (*str or int, optional*): The identifier of the data reduction/processing applied to the data.
+
+ + RAW (1)
+ + CALIBRATED (2)
+ + PRODUCT (3)
+ + AUXILIARY (0)
+
+- **data_product_type** (*str, optional*): High level description of the product.
+
+ + image
+ + spectrum 
+ + timeseries
+
+- **intent** (*str, optional*): The intent of the original obsever in acquiring this observation.
+
+ + SCIENCE 
+ + CALIBRATION
+
+- **collection** (*str, optional*): List of collections that are available in eHST catalogue.
+
+ + HLA
+ + HST
+ 
+ Do not forget that this is a list of elements, so it must be defined as ['HST'], ['HST', 'HLA']...
```

Comment:
In this case, the documentation above should be `list of strings, optional`, since a string won't be accepted